crates/sui-graphql-rpc/src/types/coin_metadata.rs

```diff
@@ -9,6 +9,7 @@ use async_graphql::*;
 
 use sui_types::coin::CoinMetadata as NativeCoinMetadata;
 
+/// The metadata for an object of type Coin.
```

Comment:
also dont know if this is rly that useful

Reply:
(Also needs to be moved to the `impl` block.